### PR TITLE
Feature/infinite scrolling

### PR DESCRIPTION
**Pull request by gilemos**
Hey guys! I did the infinite scrolling for the more options view controller. It is still slow and only goes up to 60 attractions per category, but that will be improved once I can work on Franklin's search bar. Here is how it looks like:

Video Walkthrough

### CODE REVIEW

File: Travel Scheduler/View Controllers/MoreOptionViewController.m [hunk 1]
```diff
@@ -87,7 +92,63 @@ - (void)createCollectionView
...
+        }
+        self.isMoreDataLoading = NO;
+        [loadingMoreView stopAnimating];
+        [self.collectionView reloadData];
+    }];
+}
+
+#pragma mark - Infinite scrolling helper methods
+
+- (void) setUpInfiniteScrollIndicator
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/MoreOptionViewController.m]: Uh... I don't think there should be a space after (void)?

**Review by aliu917 — APPROVED** [review]
Great work! Everything looks good.

File: Travel Scheduler/View Controllers/MoreOptionViewController.m (hunk 1 above)
Comment (gilemos) [reply Travel Scheduler/View Controllers/MoreOptionViewController.m]: You're right! Gonna fix that

Comment (gilemos) [reply Travel Scheduler/View Controllers/MoreOptionViewController.m]: Done!

**Pull request merged**